data/hai/hai reveal 4 middle.txt

```diff
@@ -235,21 +235,29 @@ mission "Hai Rescue [A04] Tarazed Analysis"
+			`	"Here, let me take a look at what you've brought me." He takes the data card you got from the Quarg and plugs it into the communicator. After several minutes of typing and muttering incoherently to himself he pauses as a simulation plays on his screen. You're not sure exactly what you're looking at but when it finishes he lets out a long slow breath and comments to himself in a low voice. "Bastijn Biesterveld you absolute legend. He was right, it's the interference."`
+			`	He turns back to you and explains. "Normally interference isn't something that's logged, but when you're in civilized space with hyperspace relays as your ship jumps it does a final handshake with the local relay station. That station sends that handshake signature to the destination station so it knows exactly who you are to be able to reconnect communications at the other end. It's a fully automatic process and it's independent of ship ID's, it's just buried in there as a check from back in the days when hyperdrives would occasionally explode and we'd need to work out who went missing. It gets logged and automatically overwritten every couple days, and someone would probably have gotten rid of it if it wasn't so deeply baked into the way the system works. If you know when a ship jumped and where it jumped to, you can use it to confirm it didn't blow up; and if there's something wrong with the communications equipment you can use the log info to identify if a specific ship is the same ship talking to you later, but neither of those are particularly useful since the same information can be acquired more easily and in much higher quality through other means."`
+			`	He stops and looks sheepish for a moment. "Sorry, that was probably way more explanation than you needed. There's a small group of us comms people that have lost people or know people who have, that the authorities can't help. We're scattered all over and we've been looking at this ourselves. Together we identified some ships we thought were suspicious but we've not been able to pin down anything concrete. Bastijn was convinced there was a pattern of interference signatures from the logs; he made the analysis program but it was never definitively distinct from the noise. Now though, well what you've given me is a match!"`
+			`	He casts around for a moment and then plugs in a freshly unpacked data card into his console. "Gimme a minute I can do something for you now. Just uh, sit down for ten minutes and I'll be back with you."`
```

Comment:
```suggestion
			`	He casts around for a moment and then plugs in a freshly unpacked data card into his console. "Give me a little time: I can do something for you now. I'll be back with you in ten minutes."`
```
The dialogue style suddenly changed for this line when compared to what's before it.
Also, what's `casts` mean? Don't quite understand how it's being used here.

Reply:
"To look around for something" - but it connotes looking in a manner where you're mostly looking in your mind for where you expect to find a thing as opposed to physically using your eyes to identify something.
e.g. when you're mentally rifling through the draws of your desk to recall which draw you put something in, or scanning a messy table in the full knowledge that what you're looking for is probably covered by something but with visual context clues you can retrace your steps mentally to recall where you left it.

https://dictionary.cambridge.org/dictionary/english/cast-around

In this context it implies he's either a) working out where he left his fresh drives to get one, b) thinking where he might have an unused data port to plug something in to - assuming he's using most of his ports semi-permanently, or c) both.

Reply:
Gotcha, thanks! I'm good with it here then.